client_test.go

```diff
@@ -111,22 +228,18 @@ func TestClientConfig(t *testing.T) {
 }
 
 func TestClientReconnect(t *testing.T) {
-	client := NewClient(clientTestURL).WithDialConfig(amqp.Config{
-		Properties: amqp.Table{
-			"connection_name": "client-reconnect-test",
-		},
-	})
-	defer client.Stop()
+	_, client, start, stop := initTest()
+	start()
 
-	assert.NotNil(t, client, "client with dialer exist")
+	defer stop()
 
-	// Force a connection by calling send.
-	_, err := client.Send(NewRequest().WithResponse(false))
-	assert.NoError(t, err)
+	closeConnections(testClientConnectionName)
+	time.Sleep(1 * time.Second) // Default wait for reconnect is 0.5s.
```

Comment:
Thiis looks an awful lot like 1 second. 😃

Reply:
Heh, I think we wait 1 second so that the reconnect has had enough time.

Reply:
My comment was related to the fact that the code comment says '0.5s' if it wasn't obvious. :)

Reply:
I'll make it more clear :)